component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -875,7 +876,7 @@ public List<Property> getConfigurationProperties() {
         payload.setDisplayName("HTTP Payload");
         payload.setRequired(false);
         payload.setDescription("Enter the HTTP Payload used by the SMS API. If the phone number and text message are " +
-                "in Payload, specify them as $ctx.num and $ctx.msg");
+                "in Payload, specify them as $ctx.num and $ctx.msg or $ctx.otp");
```

Comment:
Same as stated previously, shall we change the description to something like follows.
```
payload.setDescription("Enter the HTTP Payload used by the SMS API. If the phone number, text message, and OTP are " +
                "in Payload, specify them as $ctx.num, $ctx.msg, and $ctx.otp");
```